src/back/spv/writer.rs

```diff
@@ -744,12 +744,12 @@ impl Writer {
             crate::TypeInner::Scalar { kind, width } => {
                 instruction = match kind {
                     crate::ScalarKind::Sint => {
-                        self.instruction_type_int(id, width, Signedness::Signed)
+                        self.instruction_type_int(id, width << 3, Signedness::Signed)
```

Comment:
It would be cleaner to define a constant like `BITS_PER_BYTE = 8` and use `* BITS_PER_BYTE` here. Note that LLVM will convert it to a bit shift for us, anyway.